dev_tools/auto_merge.py

```diff
@@ -509,7 +508,7 @@ def attempt_sync_with_master(pr: PullRequestDetails
     data = {
         'base': pr.branch_name,
         'head': master_sha,
-        'commit_message': 'Update branch (automerge)'.format(pr.branch_name)
+        'commit_message': 'Update branch (automerge)'
```

Comment:
The original looks like it was accidentally missing a {}. Suggestion:

```python
'commit_message': 'Update branch {pr.branch_name} (automerge)'
```

Reply:
No, it was intentional. When I changed it to not mention the branch name I just forgot to remove the format call.